docs/CONTRIBUTING.md

```diff
@@ -23,6 +23,20 @@ Once you have cloned the JsDbg repository, please follow the steps below to buil
       * Common build errors and fixes:
          * Missing Windows 10 SDK error - install the correct SDK version (should be specified in the error message) and rebuild the solution.
 
+### Building JsDbg (Linux)
+
+1. Get the [x64 .NET Core SDK binary](https://dotnet.microsoft.com/download) (click the x64 link in the left column in the table at the bottom). Instructions below and `create_package.sh` assume you extract it into `~/dotnet`.
```

Comment:
Do you mean to say "Follow the instructions below"?

Reply:
Also, not sure what you mean by "assume you extract it into ..."

Reply:
I meant that the instructions were written with an assumption that dotnet is extracted into ~/dotnet. I've rewritten this a bit.